api.js

```diff
@@ -0,0 +1,47 @@
+const express = require("express")
+const router = express.Router()
+const urllib = require('urllib')
+
+
+  const teamToIDs = {
+    "lakers": "1610612747",
+    "warriors": "1610612744",
+    "heat": "1610612748",
+    "suns": "1610612756"
+}
+
+const json = {
```

Comment:
why do you need a "json" variable and inside it "data"? you could just define - data = {} and use that...